tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -343,3 +347,68 @@ def test_unreachable_code_sections(
     (i.e. code sections not reachable from the code section 0).
     """
     eof_test(data=container, expect_exception=EOFException.UNREACHABLE_CODE_SECTIONS)
+
+
+@pytest.mark.parametrize("callee_outputs", [1, 2, MAX_CODE_OUTPUTS])
+def test_callf_stack_height_limit_exceeded(eof_test, callee_outputs):
+    """Test for invalid EOF code containing CALLF instruction exceeding the stack height limit."""
+    callf_stack_height = 1024 - callee_outputs
+    container = Container(
+        sections=[
+            Section.Code(
+                Op.PUSH0 * callf_stack_height + Op.CALLF[1] + Op.POP + Op.STOP,
+                max_stack_height=1024,
+            ),
+            Section.Code(
+                Op.PUSH0 * callee_outputs + Op.RETF,
+                code_outputs=callee_outputs,
+                max_stack_height=callee_outputs,
+            ),
+        ],
+    )
+    eof_test(data=container, expect_exception=EOFException.MAX_STACK_HEIGHT_ABOVE_LIMIT)
+
+
+@pytest.mark.parametrize("callee_outputs", [1, 2, MAX_CODE_OUTPUTS])
+def test_callf_stack_overflow_by_outputs(eof_test, callee_outputs):
+    """
+    Test for invalid EOF code containing CALLF instruction exceeding the runtime stack height limit
+    by calling a function with non-zero outputs.
+    """
+    callf_stack_height = 1025 - callee_outputs
```

Comment:
It would help to expand that 1025 is runtime max stack limit of 1024 (not to confuse with `MAX_OPERAND_STACK_HEIGHT`, which is 1 less) plus one.

Reply:
Sadly there is no constant for the runtime stack height limit (1024).

Reply:
Added constant.